nltk/corpus/reader/verbnet.py

```diff
@@ -70,30 +71,32 @@ def __init__(self, root, fileids, wrap_etree=False):
     """Regular expression used by ``_index()`` to quickly scan the corpus
        for basic information."""
 
-    def lemmas(self, classid=None):
+    def lemmas(self, vnclass=None):
```

Comment:
I mean the usage example in the format of docstrings. Something like https://github.com/nltk/nltk/blob/develop/nltk/parse/dependencygraph.py#L146